frontend/src/core/codemirror/extensions.ts

```diff
@@ -86,10 +87,9 @@ export function scrollActiveLineIntoView() {
       // Only scroll if there is an active line
       if (activeLines.length === 1) {
         const activeLine = activeLines[0] as HTMLElement;
-        smartScrollIntoView(activeLine, {
-          top: 30,
-          bottom: 120,
-        });
+        const appEl = document.getElementById("App");
+        invariant(appEl, "App not found");
+        smartScrollIntoView(activeLine, { top: 30, bottom: 150 }, appEl);
```

Comment:
this was a bug that i regressed when adding the app footer a while back. this makes it so your last line it not at the bottom, but rather 150px from the bottom